packages/mongodb/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.8.0"
+  changes:
+    - description: Update field yml files
```

Be more explicit on the change done, for example:
```suggestion
    - description: Fix type of the `mongodb.replstatus.member.primary.optime` field.
```

---

Done